examples/alice_chopper/lib/main.dart

```diff
@@ -35,11 +36,8 @@ class MyApp extends StatefulWidget {
 class _MyAppState extends State<MyApp> {
   final AliceChopperAdapter _aliceChopperAdapter = AliceChopperAdapter();
 
-  late final Alice _alice = Alice(
-    showNotification: true,
-    showInspectorOnShake: true,
-    maxCallsCount: 1000,
-  )..addAdapter(_aliceChopperAdapter);
+  late final Alice _alice = Alice(configuration: AliceConfiguration())
```

Comment:
Since this is an example, I'd maybe put the configuration declaration on a separate line, just so that it's as explicit as possible for a new user, i.e.
```dart
final AliceConfiguration _conf = AliceConfiguration();

late final Alice _alice = Alice(configuration: _conf);
```

Reply:
👍